docs/source/welcome_to_sycamore/get_started.md

```diff
@@ -107,25 +110,25 @@ docker compose run sycamore_crawler_http <url>
 # docker compose run sycamore_crawler_http http://www.aryn.ai
 ```
 
-This will crawl and download the data from the specified website. If you import aryn.ai, you can try "who are the Aryn founders?"
+This will crawl and download the data from the specified website(s).
```

Comment:
Do you want the (s)
It will only crawl from *.aryn.ai in the example, it won't go out of the domain.

Reply:
Ack